views/css/adminOrderView.css

```diff
@@ -300,3 +300,34 @@
 .modal.ps-checkout-refund.legacy .modal-content-container{
   padding: 10px
 }
+
+.checkout-modal-container {
+  padding: 100px 180px;
```

Comment:
Adjust pading on smaller screens, like on <=992px screens apply 16px padding to maximize space usage